codegen/server/client_cli.go

```diff
@@ -0,0 +1,691 @@
+package server
```

Comment:
More I think about it, we can move this into a different package called `cli` which lives in `codegen/cli` because everything here is CLI specific. That will also save you some refactor headaches.